src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/MsftToSbSdkFiles.diff

```diff
@@ -45,9 +45,9 @@ index ------------
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/
  ./packs/Microsoft.NETCore.App.Ref/x.y.z/analyzers/
 @@ ------------ @@
- ./packs/NETStandard.Library.Ref/x.y.z/ref/netstandard2.1/System.Xml.XPath.XDocument.dll
  ./sdk-manifests/
  ./sdk-manifests/x.y.z/
+ ./sdk-manifests/x.y.z/
```

Comment:
Version number abstraction makes this difficult to understand.  There must be three sdk-manifest version folders right now in the MSFT build?

Reply:
Oh, good point. I was confused by what this actually meant so thanks for your comment. So there are two subdirectories now:
* 8.0.100-rc.1
* 8.0.100-rc.2

This seems to just be a build coherency issue then. In that case, I don't think it's worth updating the baseline since it will just need to be reverted with this is resolved.